Quickstarts/DotNet/App.config

```diff
@@ -1,34 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The *.suo, *.exe should be removed from the repo, You should get better .gitignore.